kpi/utils/mongo_helper.py

```diff
@@ -292,7 +292,11 @@ def _get_cursor_and_count(cls, mongo_userform_id, hide_deleted=True,
             # Retrieve all fields except `cls.USERFORM_ID`
             fields_to_select = {cls.USERFORM_ID: 0}
 
-        cursor = settings.MONGO_DB.instances.find(query, fields_to_select)
+        cursor = settings.MONGO_DB.instances.find(
+            query,
+            fields_to_select,
+            max_time_ms=MONGO_DB_MAX_TIME_MS
```

Comment:
I believe it's `settings.MONGO_DB_MAX_TIME_MS` isn't it? ;-)

Reply:
Yes, it is 😨 
It's fixed